pkg/hubble/cell/cell.go

```diff
@@ -35,11 +37,18 @@ var Cell = cell.Module(
 	"hubble",
 	"Exposes the Observer gRPC API and Hubble metrics",
 
-	cell.Provide(newHubbleIntegration),
-	cell.Config(defaultConfig),
+	Core,
 
 	// Hubble flow log exporters
 	exportercell.Cell,
+
+	// Parser for Hubble flows
+	parsercell.Cell,
+)
+
+var Core = cell.Group(
```

Comment:
Nit: I suggest to add a comment to explain the purpose of this group (we may also add an example of how it can be used to have a "customized" hubble, similar to the one in [this comment](https://github.com/cilium/cilium/pull/38368#discussion_r2007534285))